velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -95,33 +111,37 @@ class MultiNthValueTest : public NthValueTest,
   MultiNthValueTest() : NthValueTest(GetParam()) {}
 };
 
-// Tests nth_value with data of a uniform distribution.
+// Tests (nth|first|last)_value functions with data of a uniform distribution.
```

Comment:
Instead of repeating these comments for all test methods, can we add a comment to the MultiNthValueTest class instead?

Reply:
Thanks for the suggestion, added a single comment summing up the different dataset types that are tested to ```NthValueTest::testValueFunctions(const std::vector<RowVectorPtr>& input)```.